### ISSUE TITLE
Local media assigned to"Various Artists" displays no tracks in track view

### ISSUE THREAD

**Issue by FrankJaco** [music-assistant/hass-music-assistant#748]
### What version of Music Assistant has the issue?

2022.7.1

### The problem

Local media characterized as "Various Artist" displays no tracks in track view. 

If you open the "Various Artist" selection...
![image](https://user-images.githubusercontent.com/79318542/181359964-fd22113d-910c-475f-b9b4-0f4a1dad5b96.png)
you will see the message "no items found"...
![image](https://user-images.githubusercontent.com/79318542/181360281-273384b9-fa35-4a8c-886e-f1e4c42969c0.png)

If you select "Albums" it displays as expected and the tracks are available under Albums.
![image](https://user-images.githubusercontent.com/79318542/181360513-887ea361-7ec4-4095-8252-403523501f6b.png)

It is conceivable this is all by design, but if so the message in "Tracks" should be something informative, not "no items found".

### How to reproduce

As shown above.

### Relevant log output

not applicable

### Additional information

none

### What version of Home Assistant Core are your running

2022.7.6

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by marcelveldt** [comment]
That is a bit of an edge case with various artists as that is (in 99% of the cases) an album artist only...
Shall I just hide the entire tracks tab in this case?

**Comment by FrankJaco** [comment]
Yes, I would this is a very minor issue.  And your proposed solution is 
a good one.... It should default to "Albums" when you first open 
"Various artists" up, and of course still allow track view once you 
navigate to Albums.

fj

PS:  If I haven't said it enough....   Keep up the great work. This is 
going to be one of the best Home Assistant additions period.

**Comment by OzGav** [comment]
This appears to depend on the sources you have. In my case I see some Spotify and YouTube tracks (but no local tracks as reported)
![image](https://user-images.githubusercontent.com/19848947/181485003-a76ffe39-fba8-4d78-af8b-4e1820ed84d1.png)

**Comment by OzGav** [comment]
But to be honest I would prefer all artists to open in album view instead of tracks view but thats just me ;-)

**Comment by mromaithi** [comment]
guys how can I add the local folders to music assistant? what the mount point?
thx

**Comment by FrankJaco** [comment]
MA using the default location for Home Assistant:   the 'media' 
directory.  Mounting a outside directory is possible, but challenging 
and differs depending on your "flavor" of Home Assistant. (So 
challenging I never got a mount point working to my satisfaction.) I 
personally run HA on a micro form factor PC with the Hass OS.  I have 
two SSD's: 128GB M.2 and a 1TB SATA3.   I used the "move datadisk" 
procedure to move "media" (and other HA directories) to the 1TB SSD and 
have created a "music" directory inside /media and in the process of 
moving about 300GB's of MP3's there.

Oh, I use the Samba Addon in Home Assistant to give me direct access to 
the media (and other HA directories) so that I can easily move files in 
and out. It actually is possible, but cumbersome to add files to the 
media directory via the HA gui: (Media in left menu bar, then go to 
Local Media).

fj

**Comment by OzGav** [comment]
Please read the FAQ here https://github.com/music-assistant/hass-music-assistant/discussions/710 The section you want is called “ How do I access my local music / How do I tag my local music”

**Comment by OzGav** [comment]
In relation to the OP this is now being tracked in the MA 2 backlog so this will be fixed soon and I will close this.
